src/components/Permalink/Permalink.js

```diff
@@ -63,9 +67,9 @@ class Permalink extends PureComponent {
       dispatchSetZoom(z);
     }
 
-    const lineFilter = parameters[LINE_FILTER];
-    const routeFilter = parameters[ROUTE_FILTER];
-    const operatorFilter = parameters[OPERATOR_FILTER];
+    const lineFilter = getUrlParam(parameters[LINE_FILTER]);
+    const routeFilter = getUrlParam(parameters[ROUTE_FILTER]);
+    const operatorFilter = getUrlParam(parameters[OPERATOR_FILTER]);
```

Comment:
You don' manage the case insentivity

Reply:
```suggestion
    const operatorFilter = getUrlParam(parameters[OPERATOR_FILTER.toLowerCase()]);
```

Reply:
works as expected with the current version: 
'publishedlinename=s1, s2, S 15, S  9'

Reply:
But needs this PR to be merged first: https://github.com/geops/react-transit/pull/34